pkg/controller/clusterstate/cache/cache_test.go

```diff
@@ -199,3 +202,31 @@ func TestAddPod(t *testing.T) {
 	}
 }
 
+func TestNodeStatus(t *testing.T) {
+
+	var memory float64 = 0.0
+	var millicpu float64 = 0.0
+	var gpu int64 = 0.0
+
+	node1 := buildNode("n1", buildResourceList("2000m", "10G"), string(v1.NodeDiskPressure))
+
+	cache := &ClusterStateCache{
+		Nodes: make(map[string]*api.NodeInfo),
+	}
+
+	cache.addNode(node1)
```

Comment:
If you add an "unllocatable node" to this test, it will be "LGTM" :)

Reply:
Node with status v1.NodeDiskPressure is unallocatable according to MCAD, do we need another test and if yes what do we think should be the node condition?

Reply:
Yes, to both questions. I would try to get close to 100% coverage for this test case.

Reply:
v1. NodeDiskPressure is already part of this PR